src/Sulu/Bundle/AdminBundle/Tests/Unit/Metadata/FormMetadata/StructureFormMetadataLoaderTest.php

```diff
@@ -35,7 +35,7 @@ class StructureFormMetadataLoaderTest extends TestCase
     /**
      * @var string
      */
-    public const CACHE_DIR = __DIR__ . '/../../../../../../../tests/Resources/cache';
+    public const CACHE_DIR = __DIR__ . '/../../../../../../../../tests/Resources/cache';
```

Comment:
this class was moved and the path was not adopted which did end in creating a `src/tests` directory instead of resources/cache dir.